memberlist.go

```diff
@@ -198,6 +203,8 @@ func newMemberlist(conf *Config) (*Memberlist, error) {
 		}
 	}
 
+	metricLabels := conf.MetricLabels
```

Comment:
nit: why make a local variable for two uses below?

Reply:
oops, I forgot to delete the variables after change the type; we used to have `metricLabels := mapToLabels(conf.MetricLabels)`

Fixed now.